### DISCUSSION TITLE
Support non-english locales for default strings - g11n/i18n

### DISCUSSION THREAD

**Discussion by hermansb** [carbon-design-system/carbon#8729] [Ideas]
Default strings such as `Expand`/`Collapse` [here](https://github.com/carbon-design-system/carbon-components-react/blob/c3bf0123a2a98ca84acfc7e86e69840eaf57a89e/src/components/Tile/Tile.js#L352-L353) do not support a `locale` property in order to use languages other than English as a default string.

It is possible to pass in your own translated strings to override the English default via the `tileCollapsedIconText` and `tileExpandedIconText` properties [for the react components anyway] and for those that want to keep the defaults, they can pass in localized versions of the defaults in this way.

However, for those consumers in this position it would be most convenient to just have the default strings also provide defaults for the standard locales.

**Comment by jnm2377** [comment]
Hey @hermansb thanks for opening this discussion. This seems like a fair ask. I'm opening an issue from this so we can keep track of it in our backlog. If you'd like to contribute a PR to fix this, we'd welcome it, otherwise we'll get to it when we have the time. :)
